packages/libpluto/build.sh

```diff
@@ -0,0 +1,49 @@
+TERMUX_PKG_HOMEPAGE=https://plutolang.github.io/
+TERMUX_PKG_DESCRIPTION="Shared library for the Pluto interpreter"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="Komo @cattokomo"
+TERMUX_PKG_VERSION="0.9.4"
+TERMUX_PKG_SRCURL=https://github.com/PlutoLang/Pluto/archive/refs/tags/${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=6a4fae052a9ad47e29a6767047c3e55ab5c887486d14e39248657ff43b45875c
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_DEPENDS="libc++"
+TERMUX_PKG_BUILD_DEPENDS="readline"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_BREAKS="plutolang (<< 0.9.4-2)"
+TERMUX_PKG_REPLACES="plutolang (<< 0.9.4-2)"
+
+# can't use php version from termux repo, there's no build for the version
```

Comment:
The script fetches the pre-built static PHP binary from <https://static-php.dev>, which doesn't have pre-built for version 8.3.10.
Although it can be resolved by building from source using `spc` from <https://static-php.dev>, but it will require another patches to get it working.

Reply:
Honestly, the scripts would work fine as long as it doesn't contain a syntax from previous/next version that doesn't exist in pinned version. There's no need to pin to latest version according to `php` package version, unless it is required to do so.

Reply:
I didn't realize this is another unofficial build of PHP.
Not pinning it to the PHP package's version is gonna be fine.
I just wanted to keep it synced so the version used to build plutolang doesn't drift out of sync with the PHP package.